docs/news/2020-12-03-Dockstore-1.10.0.rst

```diff
@@ -0,0 +1,47 @@
+Dockstore 1.10.0
+================
+
+Highlighted new features include:
+---------------------------------
+
+-  GitHub app support for filters to allow for more control over which tags and branches are automatically reported to Dockstore.
+
+   -  More details `here <https://docs.dockstore.org/en/develop/getting-started/github-apps/github-apps.html?highlight=filters>`_.
```

Comment:
Sorry, I missed this the first review, but this should be a relative link, especially since we're talking about moving away from develop as the publicly available docs.

Sort of reluctant to bring this up, and may not be worth the effort, but we do try to avoid absolute paths